### ISSUE TITLE
[question] Format for community device plugins

### ISSUE THREAD

**Issue by CarbonCollins** [hashicorp/nomad#9806]
Hi, I plan on adding a USB plugin to the [community device plugins docs page](https://www.nomadproject.io/docs/devices/community). 

What is the intended format for this page? Should it be a markdown table on the community page itself or should it be a sub page similar to the [Task Drivers](https://www.nomadproject.io/docs/drivers)?

**Comment by tgross** [comment]
Hi @CarbonCollins! That's exciting to see!

For the contents, I'd use the [Nvidia device plugin](https://www.nomadproject.io/docs/devices/nvidia) as a guide for the sections you'll want: the fingerprinted attributes, installation guide, and plugin configuration at a minimum, and an example jobspec would be great.

For the structure of the file tree, let's use the existing `drivers` as an example. So if we look at the [drivers](https://github.com/hashicorp/nomad/tree/master/website/content/docs/devices) directory, we'll add an `external` directory, and within that have `collins-usb.mdx` (or whatever the name of the plugin is).

There's also a [javascript file](https://github.com/hashicorp/nomad/blob/77c4cff62281796032782382d0eb1e7f442ef80f/website/data/docs-navigation.js#L374-L401) that generates the sidebar navigation which will need an updated section:

```js
{
  category: 'devices',
  content: [
    'nvidia',
    {
      category: 'external'
      content: [
        'collins-usb'
      ]
    }
  ]
},
```

But honestly if you can write up the page whoever on the team reviews it would be happy to do the extra bit of legwork to get it into the navigation.

**Comment by tgross** [comment]
Forgot to circle back and close this issue once #9879 got merged. That'll get published to the website once 1.0.4 goes live, btw.
